src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -287,6 +290,33 @@ public IActionResult GetAdminToken()
             return Ok(requestHeaderToken);
         }
 
+        [HttpPost]
+        [Route("admin/warmup")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        [RequiresRunningHost]
+        public async Task<IActionResult> Warmup([FromServices] IScriptJobHost scriptHost, [FromServices] IScriptHostManager scriptHostManager, [FromServices] IHostIdProvider hostIdProvider)
+        {
+            // The function must be called warmup and should be of type warmup trigger
+            FunctionDescriptor function = scriptHost.GetFunctionOrNull(WarmUpConstants.FunctionName);
+            if (function != null && function.Metadata.ContainsInputBinding(WarmUpConstants.WarmupTrigger))
```

Comment:
Change the extension method to filter on triggers only and rename the method to something like `HasTrigger` taking the trigger type name, or `HasTriggerType`

Reply:
This is not relevant anymore as moved to another extension.